core/domain/exp_services.py

```diff
@@ -1461,17 +1461,18 @@ def accept_suggestion(editor_id, thread_id, exploration_id, commit_message):
         suggestion = feedback_services.get_suggestion(
             exploration_id, thread_id)
         suggestion_author_username = suggestion['author_name']
-        full_commit_message = ('Accepted suggestion by %s: %s' % (
-            suggestion_author_username, commit_message))
         change_list = _create_change_list_from_suggestion(suggestion)
+        suggestion_author_id = user_services.get_user_id_from_username(
+            suggestion_author_username)
         update_exploration(
-            editor_id, exploration_id, change_list, full_commit_message)
+            suggestion_author_id, exploration_id, change_list,
```

Comment:
Hang on -- shouldn't the author id here be the editor_id (since they're actually making the commit), with the author username prepended to the commit message?

I wonder if, in cases like this, we should also store the suggestion_author_id in a separate field in the commit history. What do you think?